scripts/leaderboard_backend/build_script.sh

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/bash
+
+cd ../lm-evaluation-harness
+pip install -e .
```

Comment:
This should be done through requirements.txt or any other way for specifying dependence

Reply:
There is a problem with this, because in order for proper validation through lm evaluation harness on goat dataset, I need to build lm evaluation harness that is in deepvk repository (since that repository has support for goat tasks), that's why I build it through  ```pip install -e .``` command from the lm evaluation harness directory. Maybe you have some ideas how I can improve this approach?